Makefile

```diff
@@ -245,7 +248,10 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 		kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
 .PHONY: deploy
-deploy: install ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+deploy: install cmctl ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	kubectl apply --server-side --force-conflicts -f \
+		https://github.com/jetstack/cert-manager/releases/download/v$(CERTMANAGER_VERSION)/cert-manager.yaml
```

Comment:
Imagine running this on k8s with a cert-manager already installed and configured? Can this potentially break their system? 

IMO, we should first validate if certmanager exists and install it only if there isn't any. 
Additionally, consider exiting with error if cert-manager is absent. 
We can also provide `install-certmanager` helper to make life easier for vanilla k8s installation.